cpp/ps0004/ps0004_sol.cpp

```diff
@@ -3,10 +3,14 @@
 using namespace std;
 
 int main() {
-    int a;
+    // prompting user for number 1/1.5
+    float a;
     cout << "Enter a number from 32 to 126: ";
     cin >> a;
-    if (a % 1 != 0) {
+

```

Comment:
- [x] declared a variable to store the user input
- [ ] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input